packages/test/test-end-to-end-tests/src/test/sharedStringEndToEndTests.spec.ts

```diff
@@ -188,7 +188,11 @@ describeCompat("SharedString grouped batching", "NoCompat", (getTestObjectProvid
 		await provider.ensureSynchronized();
 	});
 
-	it("can load summarized grouped batch", async () => {
+	it("can load summarized grouped batch", async function () {
+		// We've seen flakiness in ODSP and r11s. This test is verifying SharedString logic regardless of what service handles the ops/summary.
+		if (!["local", "tinylicious", "t9s"].includes(provider.driver.type)) {
```

Comment:
I've been curious - Why do we need to include both spellings of t9s?

Reply:
The `TestDriverTypes` unfortunately has `t9s` and `tinylicious` as well as `r11s` and `routerlicious`. Technically the npm script will only use one of those values, but better to be safe.
Ideally, we'd only have one of these values for each corresponding driver as it's redundant to have both.